components/more_direct_channels/last_post_at.tsx

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React, {FunctionComponent} from 'react';
+import moment from 'moment-timezone';
+import {FormattedMessage} from 'react-intl';
+
+type LastPostAtProps = {
+    lastPostAt?: number;
+}
+import {t} from 'utils/i18n';
+
+export const LastPostAt: FunctionComponent<LastPostAtProps> = ({lastPostAt}: LastPostAtProps): JSX.Element | null => {
+    if (!lastPostAt) {
+        return null;
+    }
+    let messageId: string;
+    let value: number;
+
+    if (moment().diff(lastPostAt, 'years') > 0) {
+        messageId = t('more_direct_channels.last.post.at.years');
+        value = moment().diff(lastPostAt, 'years');
+    } else if (moment().diff(lastPostAt, 'months') > 0) {
+        messageId = t('more_direct_channels.last.post.at.months');
+        value = moment().diff(lastPostAt, 'months');
+    } else if (moment().diff(lastPostAt, 'days') > 0) {
+        messageId = t('more_direct_channels.last.post.at.days');
+        value = moment().diff(lastPostAt, 'days');
+    } else if (moment().diff(lastPostAt, 'hours') > 0) {
+        messageId = t('more_direct_channels.last.post.at.hours');
+        value = moment().diff(lastPostAt, 'hours');
+    } else {
+        messageId = t('more_direct_channels.last.post.at.minutes');
+        const minutes = moment().diff(lastPostAt, 'minutes');
+        value = minutes || 1;
+    }
+
+    return (
+        <div className='more-modal__last_post_at'>
+            <FormattedMessage
```

Comment:
Love the relative time! Really makes it obvious that they're recent conversations.

React-intl's `<FormattedRelativeTime>` could be employed here as it has built-in i18n. That might save a bit of effort on the translations front? 

I'm really interested in this as I'm working on/almost done with a `<Timestamp/>` [component](https://github.com/mattermost/mattermost-webapp/blob/f7b8fdfdc1d5df590d7865d6beb45b1983965ab2/components/timestamp/timestamp.tsx) for another PR (#5390). Maybe we could combine efforts a bit—would love your input!